starfish/test/full_pipelines/cli/_base_cli_test.py

```diff
@@ -0,0 +1,54 @@
+import os
+import shutil
+import subprocess
+import tempfile
+import unittest
+from typing import List, Tuple
+
+from starfish.intensity_table import IntensityTable
+from starfish.util import clock
+
+
+class CLITest(unittest.TestCase):
+    __test__ = False
```

Comment:
Can you add a comment about what this does (and that you have to make it true for the test to work, based on what I think I'm seeing below)?